knitr/curse-dims/curse-dims.Rmd

```diff
@@ -0,0 +1,770 @@
+is the mode (highest density point) of the 16-dimensional unit normal
+distribution.  But just how much less density do the draws have than
+the maximum density?  The following plot of the median log density and
+99% itervals along with the density at the mode illustrates.
+
+```{r}
+plot2 <-
+  ggplot(df, aes(dim)) +
+  geom_ribbon(aes(ymin = lb_ll, ymax = ub_ll), fill="lightyellow") +
+  geom_line(aes(y = mid_ll)) +
+  geom_point(aes(y = mid_ll)) +
+  geom_line(aes(y = max_ll), color="red") +
+  geom_point(aes(y = max_ll), color="red") +
+  scale_x_log10(breaks=c(2^(0:(D/2)))) + 
+  scale_y_continuous() + 
+  ylab("log density") +
+  xlab("number of dimensions") +
+  ggtitle("Draws have Much Lower Density than the Mode\n (median and 99% intervals; mode in red)");
+plot2;
+```
+
+## 5. Typical Sets
+
+Roughly speaking, the typical set is where draws from a given
+distribution tend to lie.  That is, the typical set encompasses almost
+all of the probability mass for a distribution (sets of outcomes for
+discrete distributions and volumes for continuous ones).  What we will
+see in this section is that the typical set is usually nowhere nearthe
+mode of the distribution.
+
+#### A Discrete Example of Typicality
+
+This is easy to see with a binomial example.  Consider a binary trial
+with an eighty percent chance of success (i.e., draws from
+$\mathsf{Bernoulli}(0.80)$).  Now consider repeating such a trial one
+hundred times, drawing $y_1, \ldots, y_{100}$ independently according
+to $y_n~\sim~\mathsf{Bernoulli}(0.8)$.  We would expect somewhere
+around 80 successes in such a situation.
+
+What is the highest probability sequence?  The sequence with 100
+successes in a row.  It has probability $0.8^{100}$, whereas the
+probability of any given sequence with 80 successes is only $0.8^{80}
+\, 0.2^{20}$; the sequence with 100 successes is a whopping $4^{20}$,
+or about $10^{12}$ times more probable than any specific sequence with
+80 successes and 20 failures.  But there are a lot of sequences with
+80 successes and 20 failures---a total of $\binom{100}{20}$ of them to
+be exact (around $10^{20}$).  So even though any given sequence of 80
+success and 20 failures is improbably, there are so many such
+sequences that their overall mass is higher than that of the sequence
+with 100 success.  
+
+The binomial distribution is the distribution of counts, so that if
+$y_1, \ldots, y_N$ is such that each $y_n \sim
+\mathsf{Bernoulli}(\theta)$, then 
+
+\[
+(y_1 + \cdots + y_N) \sim \mathsf{Binomial}(N, \theta).
+\]
+
+The binomial aggregates the multiple trials by multiplying through by
+the possible ways in which $y$ successes can arise in $N$ trials,
+namely 
+
+\[
+\mathsf{Binomial}(y \, | \, N, \theta) 
+= \binom{N}{y} \, \theta^y \, (1 - \theta)^{(N - y)},
+\]
+
+where the binomial coefficient that normalizes the distribution is
+defined as the number of binary sequences of length $N$ that contain
+exactly $y$ 1 values,
```

Comment:
Just rephrased that.  And I did all the other minor typo things
you suggested.  Thanks for reading so carefully.

> On Jun 16, 2016, at 6:25 PM, Daniel Lee <notifications@github.com> wrote:
> 
> In knitr/curse-dims/curse-dims.Rmd:
> 
> > +\[
> > +(y_1 + \cdots + y_N) \sim \mathsf{Binomial}(N, \theta).
> > +\]
> > +
> > +The binomial aggregates the multiple trials by multiplying through by
> > +the possible ways in which $y$ successes can arise in $N$ trials,
> > +namely 
> > +
> > +\[
> > +\mathsf{Binomial}(y \, | \, N, \theta) 
> > += \binom{N}{y} \, \theta^y \, (1 - \theta)^{(N - y)},
> > +\]
> > +
> > +where the binomial coefficient that normalizes the distribution is
> > +defined as the number of binary sequences of length $N$ that contain
> > +exactly $y$ 1 values,
> 
> is it weird to make "1 values" a hyphenated compound noun? I had trouble parsing this the first time around.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>